src/main/java/com/minecolonies/coremod/sounds/ModSoundEvents.java

```diff
@@ -15,8 +16,18 @@
 {
     /**
      * List of sound handlers.
+     *
+     * (Making this Protected isn't viable in current state)
      */
+    @SuppressWarnings(Suppression.MAKE_PROTECTED)
     public static final List<AbstractWorkerSounds> handlers = new ArrayList<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2386)